pkg/client/inmemorycheckpointer_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2022 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package client
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInMemoryCheckpointer(t *testing.T) {
```

Comment:
This should include unit test for `CheckpointChaincodeEvent()` too, but can add that later